metricflow/sql/render/big_query.py

```diff
@@ -42,8 +44,26 @@ def render_group_by_expr(self, group_by_column: SqlSelectColumn) -> SqlExpressio
         )
 
     def visit_percentile_expr(self, node: SqlPercentileExpression) -> SqlExpressionRenderResult:
-        """Render a percentile expression"""
-        raise RuntimeError("Percentile aggregations not yet supported for BigQuery.")
+        """Render a percentile expression for BigQuery.
+
+        Use the fraction class to determine numerator and denominator for offset and quantile.
+        For example - 0.5 or median would be 1/2 - generate two quantiles and pick the 1-indexed value
+        from the result list [min, median, max].
+        """
+        if node.percentile_args.function_type is SqlPercentileFunctionType.APPROXIMATE_CONTINUOUS:
+            arg_rendered = self.render_sql_expr(node.order_by_arg)
+            params = arg_rendered.execution_parameters
+            percentile = node.percentile_args.percentile
+
+            fraction = Fraction(percentile).limit_denominator()
+
+            return SqlExpressionRenderResult(
+                sql=f"APPROX_QUANTILES({arg_rendered.sql}, {fraction.denominator})[OFFSET({fraction.numerator})]",
+                execution_parameters=params,
+            )
+        raise RuntimeError(
+            "Only approximate continous percentile aggregations are supported for BigQuery. Set use_approximate_percentile and disable use_discrete_percentile in all measures"
```

Comment:
Maybe this instead? It's a little more clear:

```suggestion
            "Only approximate continous percentile aggregations are supported for BigQuery. Set use_approximate_percentile and disable use_discrete_percentile in all percentile measures"
```

Reply:
micro-nit: line length as well:
```suggestion
            "Only approximate continous percentile aggregations are supported for BigQuery. Set "
            "use_approximate_percentile and disable use_discrete_percentile in all percentile measures"
```